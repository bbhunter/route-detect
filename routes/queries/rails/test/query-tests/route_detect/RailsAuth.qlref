route_detect/RailsAuth.ql
