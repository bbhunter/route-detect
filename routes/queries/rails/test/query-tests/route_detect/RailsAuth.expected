| app/config/routes.rb:2:3:2:43 | call to resources | Found authed Rails route, method=get path=widgets controller=widgets action=index |
| app/config/routes.rb:2:3:2:43 | call to resources | Found authed Rails route, method=get path=widgets/:id controller=widgets action=show |
| app/config/routes.rb:4:3:4:41 | call to resources | Found authed Rails route, method=get path=posts/:id controller=posts action=show |
